docs/remote-snapshotter-getting-started.md

```diff
@@ -0,0 +1,172 @@
+# Getting started with remote snapshotters in firecracker-containerd
+
+This guide will walk you through setting up firecracker-containerd to work with remote snapshotters. By the end of the guide, you should have a working firecracker-contiainerd setup that allows you to lazily load container images inside network-connected firecracker-microvms. 
+
+We support the [stargz-snapshotter](https://github.com/containerd/stargz-snapshotter) as a remote snapshotter in firecracker-containerd. The stargz-snapshotter uses a specially formatted OCI container image in a form called estargz. The estargz format adds metadata to the image which allows the stargz-snapshotter to randomly access individual files in the image over the network rather than at the layer level. This enables the stargz-snapshotter to start containers before the image is fully downloaded and lazily load additional parts of the container at run time. For more information, please read the stargz-snapshotter docs including the doc on the [estargz image format](https://github.com/containerd/stargz-snapshotter/blob/main/docs/estargz.md).
+
+# Setup
+
+## Prerequisites
+
+You will need the following to use firecracker-container with support for remote snapshotters:
+
+* A computer with a recent-enough version of Linux (4.14+), and KVM enabled
+
+* A recent installation of Docker CE (for building the firecracker root filessytem)
+
+* git
+
+* Go 1.17 or later
+
+* This repository cloned onto your local machine
+
+For more details about these requirements please see the base [getting started guide](https://github.com/firecracker-microvm/firecracker-containerd/blob/main/docs/getting-started.md)
+
+
+## Build firecracker-containerd
+
+The following commands will build firecracker-containerd (including the necessary shim and agent) as well as firecracker. It will install these binaries to `/usr/local/bin`. You do not have to install any of the components in this guide, however all of the example configuration assumes the components have been installed and will need modification if you choose to skip this step.
+
+```
+make
+make install
+make firecracker
+make install-firecracker
+```
+
+## Build a linux kernel with FUSE support
+
+We generally recommend that users download our AWS-provided linux kernel that is known to work with firecracker. Unfortunately this kernel does not support FUSE which is needed for remote snapshotters. We do provide make rules to build a working kernel with FUSE support:
+
+```
+make kernel
+make install-kernel
+```
+
+By default, this will build a linux 4.14 kernel. If you would like to use 5.10, use:
+```
+KERNEL_VERSION=5.10 make kernel
+KERNEL_VERSION=5.10 make install-kernel
+```
+
+[Note: firecracker only officially supports 4.14 and 5.10](https://github.com/firecracker-microvm/firecracker/blob/0fa080b137fd29e5bcd95073473b0a57c3868d86/docs/kernel-policy.md). 
```

Comment:
How about using one of the Firecracker's release tags rather than the raw SHA1 commit?